src/pages/clusters/Pods/components/RolloutDeployPanel.tsx

```diff
@@ -227,6 +228,18 @@ function DeployButtons({
         >
           {intl.formatMessage({ id: 'pages.pods.deployAll' })}
         </Button>
+        <Button
```

Comment:
Should the button be switched to "Cancel Auto Deploy" when entering automatic deploying mode?